examples/passport-oauth2-login/src/authentication-strategies/redirect-route.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/rest
```

Comment:
I've noticed in a few files the `Node module` comment seems to have different values:

```
// Node module: @loopback/authentication or @loopback/rest
```

Please make them all the same value appropriate for this example. See other examples. thx.

Reply:
agreed